src/main/java/org/myrobotlab/service/LocalSpeech.java

```diff
@@ -302,6 +309,33 @@ public boolean setFestival() {
     }
     return true;
   }
+  
+  
+  /**
+   * @return setFestival sets the Linux tts to festival template

```

Comment:
Should be setPico2wav. Also should actually say what the return value does, does it mean success?

Reply:
yup updated and hopefully more clear